sl_patches/spdk_export_set_thread.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/include/spdk/thread.h b/include/spdk/thread.h
+index 64b04be..50b473a 100644
+--- a/include/spdk/thread.h
++++ b/include/spdk/thread.h
+@@ -206,6 +206,13 @@ void spdk_thread_lib_fini(void);
+ struct spdk_thread *spdk_thread_create(const char *name, struct spdk_cpuset *cpumask);
```

Comment:
So are we using an API that isn't exported?   Have we reported to SPDK team?

Reply:
This API is already exported in latest SPDK. Yes, your proposal of changing DAOS looks good to me. I'll update it soon.